datasets/dutch_social/dutch_social.py

```diff
@@ -0,0 +1,237 @@
+geographical boundaries. Using natural language processing we have classified the tweets for their HISCO codes.
+If the user has provided their location within Dutch boundaries, we have also classified them to their respective
+provinces The objective of this dataset is to make research data available publicly in a FAIR (Findable, Accessible,
+Interoperable, Reusable) way. Twitter's Terms of Service Licensed under Attribution-NonCommercial 4.0 International
+(CC BY-NC 4.0) (2020-10-27)
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "http://datasets.coronawhy.org/dataset.xhtml?persistentId=doi:10.5072/FK2/MTPTL7"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = "CC BY-NC 4.0"
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URLs = {"first_domain": "https://storage.googleapis.com/corona-tweet/dutch-tweets.zip"}
+
+_LANG = ["nl", "en"]
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class DutchSocial(datasets.GeneratorBasedBuilder):
+    """
+    Annotated Covid19 tweets in Dutch language. The tweets were filtered for users who had indicated
+    their location within Netherlands or if the tweets were in Dutch language. The purpose of curating
+    these tweets is to measure the economic impact of the Covid19 pandemic
+    """
+
+    VERSION = datasets.Version("1.1.0")
+
+    # This is an example of a dataset with multiple configurations.
+    # If you don't want/need to define several sub-sets in your dataset,
+    # just remove the BUILDER_CONFIG_CLASS and the BUILDER_CONFIGS attributes.
+
+    # If you need to make complex sub-parts in the datasets with configurable options
+    # You can create your own builder configuration class to store attribute, inheriting from datasets.BuilderConfig
+    # BUILDER_CONFIG_CLASS = MyBuilderConfig
+
+    # You will be able to load one or the other configurations in the following list with
+    # data = datasets.load_dataset('my_dataset', 'first_domain')
+    # data = datasets.load_dataset('my_dataset', 'second_domain')
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="first_domain", version=VERSION, description="This part of my dataset covers a first domain"
+        )
+        # datasets.BuilderConfig(name="second_domain", version=VERSION, description="This part of my dataset covers a second domain"),
+    ]
+
+    DEFAULT_CONFIG_NAME = (
+        "first_domain"  # It's not mandatory to have a default configuration. Just use one if it make sense.
+    )
+
+    def _info(self):
+        # TODO: This method specifies the datasets.DatasetInfo object which contains informations and typings for the dataset
+        if (
```

Comment:
there's only one config so you don't need this `if` statement

Reply:
ok